flask_sse.py

```diff
@@ -135,10 +136,23 @@ def messages(self, channel='sse'):
         """
         pubsub = self.redis.pubsub()
         pubsub.subscribe(channel)
+        try:
+            for msg_dict in self.messages_loop(channel, pubsub):
+                yield Message(**msg_dict)
+        finally:
+            try:
+                pubsub.unsubscribe(channel)
+            except redis_exceptions.ConnectionError:
+                pass
+
+    def messages_loop(self, channel, pubsub):
```

Comment:
Why are you breaking this out into a separate method? I think it's fine to leave it inline.

Reply:
I agree. The separate method is for something I am still working on..... Will change.